.github/workflows/llm_performance_tests.yml

```diff
@@ -555,6 +555,7 @@ jobs:
           pip install --upgrade wheel
           pip install --upgrade omegaconf pandas
           pip install --upgrade tiktoken einops transformers_stream_generator
+          pip install --upgrade matplotlib
```

Comment:
Maybe using:
pip install --upgrade tiktoken einops transformers_stream_generator matplotlib

All of the packages in this line are extra ones for specific model generation